src/main/java/de/qabel/desktop/daemon/management/DefaultTransferManager.java

```diff
@@ -44,7 +44,7 @@ public void addDownload(Download download) {
 
     @Override
     public List<Transaction> getHistory() {
-        return Collections.unmodifiableList(history);
+        return history;
```

Comment:
to make the test more readable and because I don't know why I wrapped it unmodifiable.
The test can now call `manager.getHistory().clear();` and assert later that it is empty instead of asserting one element in the history (that is always there when you sync something because it's the root folder of the sync)